Uplift/Views/GymDetail/AdditionalInfoView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AdditionalInfoView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/22/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+/// Displays additional info from Facilities Query in GymDetailTimeInfoView
+class AdditionalInfoView: UILabel {
```

Comment:
why are we making this a `UILabel` instead of `UIView`? seems like you're adding other subviews in here